JustLog/Classes/Logger.swift

```diff
@@ -65,13 +66,17 @@ public final class Logger: NSObject {
     public var console: ConsoleDestination!
     public var logstash: LogstashDestination!
     public var file: FileDestination!
-    
+    public var custom: CustomDestination?
```

Comment:
Would you like an idea to have a collection of `CustomDestination` here instead of single object? In that case `Logger` can have as much as client wants / implements custom destinations

Reply:
I was thinking about introducing these concepts gradually, as we need them.